docs/UserGuide.md

```diff
@@ -188,11 +188,39 @@ Clears all entries of people from the SectresBook.
 
 Format: `clear`
 
-### Adding Notes `[coming in v1.2.1]`
+### Adding Notes `addNote`
 
-### Editing Notes `[coming in v1.2.1]`
+Adds a note to the SectresBook.
+
+Format: `addNote n_t/TITLE n_c/CONTENT `
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```

Comment:
I think this line might be a bit weird, when I view it in the format, the 'Tip:' part is not bolded. Other than that looks good.

Reply:
![image](https://user-images.githubusercontent.com/97170362/195991359-d3cbe7a7-c202-4564-84ee-35a7db27f1bf.png)

I followed the same format as the 'add' command,

Reply:
![image](https://user-images.githubusercontent.com/97170362/195991396-6a1dc7ce-3785-4a11-a26d-3961e0ef1b14.png)

Should look like this in the page, i think its fine.
You can resolve this once you have confirmed. Thank you!